src/main/java/org/myrobotlab/service/InverseKinematics3D.java

```diff
@@ -363,7 +374,7 @@ public void onPoint(Point point) {
     log.info("Attempting to move to {}", point);
 
     // TODO: scale / translate & rotate...
-    moveTo(point);
+    moveTo(currentArm, point);

```

Comment:
This is probably problematic.. we should make note that onPoint interface isn't aware of which arm it should be publishing points to...